Update current flank flags (#88)
runningcode/fladle: README.md

### resultsDir
The name of a unique Google Cloud Storage object within the results bucket where raw test results will be stored. The default is a timestamp with a random suffix.

### runTimeout

> The max time this test run can execute before it is cancelled. s (seconds), m (minutes), h (hours) suffixes are acceptable (default: unlimited).

Reviewer: can you add a little example here? is it like `1h2m2s` or `1h 2m 2s`? or just `1h`?

### ignoreFailedTests

> Terminate with exit code 0 when there are failed tests. Useful for Fladle and other gradle plugins that don't expect the process to have a non-zero exit code. The JUnit XML is used to determine failure. (default: false)

Reviewer: ```suggestion
Always return successful task completion even when there are failed tests. Useful when parsing JUnit XML to determine failure. (default: false)
```

### disableSharding

> Disables sharding. Useful for parameterized tests. (default: false)

Reviewer: Is this statement correct?
```suggestion
Disables sharding. All tests will run on the same device. Useful for parameterized tests which do not support sharding. (default: false)
```

### smartFlankDisableUpload

> Disables smart flank JUnit XML uploading. Useful for preventing timing data from being updated. (default: false)

Reviewer: What is "smart flank"? Can we link to docs in flank here?

### testRunnerClass

> The fully-qualified Java class name of the instrumentation test runner (default: the last name extracted from the APK manifest).

Reviewer: Is this correct?
```suggestion
The fully-qualified Java class name of the instrumentation test runner (default: the test manifest is parsed to determine the class name).
```

### localResultsDir

> Local folder to store the test result. Folder is DELETED before each run to ensure only artifacts from the new run are saved.

Reviewer: ```suggestion
Local directory to store the test results. Folder is DELETED before each run to ensure only artifacts from the new run are saved.
```

### numUniformShards

> Specifies the number of shards into which you want to evenly distribute test cases. The shards are run in parallel on separate devices. For example, if your test execution contains 20 test cases and you specify four shards, each shard executes five test cases. The number of shards should be less than the total number of test cases. The number of shards specified must be >= 1 and <= 50. This option cannot be used along max-test-shards and is not compatible with smart sharding. If you want to take benefits of smart sharding use max-test-shards instead. (default: null)

Reviewer: Looks good. Can we link to docs on "smart sharding"?

### clientDetails

> A key-value map of additional details to attach to the test matrix. Arbitrary key-value pairs may be attached to a test matrix to provide additional context about the tests being run. When consuming the test results, such as in Cloud Functions or a CI system, these details can add additional context such as a link to the corresponding pull request.

Reviewer: Can we link to gcloud docs here to better understand how these are passed to the device?
Also an example of how to consume these client details? Are these to be consumed on the test device or in scripts?

### testTargetsAlwaysRun

> Always run - these tests are inserted at the beginning of every shard useful if you need to grant permissions or login before other tests run

Reviewer: ```suggestion
Always run - these tests are inserted at the beginning of every shard. Useful if you need to grant permissions or login before other tests run
```

### otherFiles
A list of device-path: file-path pairs that indicate the device paths to push files to the device before starting tests, and the paths of files to push. Device paths must be under absolute, whitelisted paths (${EXTERNAL_STORAGE}, or ${ANDROID_DATA}/local/tmp). Source file paths may be in the local filesystem or in Google Cloud Storage (gs://…).

### networkProfile

> The name of the network traffic profile, for example LTE, HSPA, etc, which consists of a set of parameters to emulate network conditions when running the test (default: no network shaping; see available profiles listed by the `flank test network-profiles list` command). This feature only works on physical devices.

Reviewer: nice, i like the examples and the command to fetch other supported values.

### resultsDir
The name of a unique Google Cloud Storage object within the results bucket where raw test results will be stored. The default is a timestamp with a random suffix.

### runTimeout

> The max time this test run can execute before it is cancelled. s (seconds), m (minutes), h (hours) suffixes are acceptable, mixes like 1h45m are currently not supported (default: unlimited).

Reviewer: 👍

Examples:
* 20, 20s -> 20 seconds
* 30m -> 30 minutes
* 2h -> 2 hours

### ignoreFailedTests
Always return successful task completion even when there are failed tests. Useful when parsing JUnit XML to determine failure. (default: false)

### disableSharding
Disables sharding. All tests will run on the same device. Useful for parameterized tests which do not support sharding. (default: false)

### smartFlankDisableUpload
Disables smart flank JUnit XML uploading. Useful for preventing timing data from being updated. (default: false)

> [What is Smart Flank?](https://github.com/Flank/flank/blob/master/docs/smart_flank.md)

Reviewer: 👍
